File: cosky-test/src/test/java/me/ahoo/cosky/test/ClearRedisScriptsTest.java
package me.ahoo.cosky.test;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;

/**
 * ClearRedisScriptsTest .
 *
 * @author ahoo wang
 */
public class ClearRedisScriptsTest extends AbstractReactiveRedisTest {
    
    @BeforeEach
    @Override
    public void afterPropertiesSet() {
        super.afterPropertiesSet();
    }
    
    @AfterEach
    @Override
    public void destroy() {
        super.destroy();
    }
    

    @Test
    public void clear() {
        ClearRedisScripts.clear(redisTemplate, "").block();
    }
}

File: cosky-test/src/test/kotlin/me/ahoo/cosky/test/ClearRedisScriptsTest.kt
package me.ahoo.cosky.test

import me.ahoo.cosky.test.ClearRedisScripts.clear
import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

/**
 * ClearRedisScriptsTest .
 *
 * @author ahoo wang
 */
class ClearRedisScriptsTest : AbstractReactiveRedisTest() {
    @BeforeEach
    override fun afterPropertiesSet() {
        super.afterPropertiesSet()
    }

    @AfterEach
    override fun destroy() {
        super.destroy()
    }

    @Test
    fun clear() {
        clear(redisTemplate, "").block()
    }
}
